assets/js/just-track.js

```diff
@@ -13,77 +38,67 @@
     if ($('.js-newcomer-register-submit-thanks').length > 0) {
-      analytics.track('Newcomer finished Wintrgarden signup', {
-        locale: CURRENT_LOCALE
-      });
+      gaTracking(registrationCategory,
+                'finished registration',
+                'Newcomer finished Wintrgarden signup');
     }
   }
 
   function trackNewcomerSignupStart() {
-    var trackNewcomer = function(selector, regType) {
-     $(document).on('click', selector, function() {
-       analytics.track('Newcomer start Wintrgarden signup', {
-         reg_type: regType,
-         signup_url: location.href,
-         locale: CURRENT_LOCALE
-       });
-     });
-    };
-
-    trackNewcomer('.js-wintrgarden-signup-no-reg', 'No reg.');
-    trackNewcomer('.js-wintrgarden-signup-reg', 'Reg.');
+    var trackNewcomer = function(selector, eventLabel) {
+      $(document).on('click', selector, function() {
+        gaTracking(registrationCategory,
+                   'started',
+                   eventLabel);
+      });
+    }
+
+    var newcomerStart = 'Newcomer start Wintrgarden signup';
+    trackNewcomer('.js-wintrgarden-signup-no-reg', newcomerStart + ' - No reg.');
+    trackNewcomer('.js-wintrgarden-signup-reg', newcomerStart + ' - Reg.');
   }
 
   function trackCompanySignup() {
     $('.js-company-signup-form').submit(function() {
-      var formData = $(this).serializeArray();
-
-      var email = findValueWithName(formData, 'email');
-      var phone = findValueWithName(formData, 'phone');
-      var name = findValueWithName(formData, 'last_name');
-      var company = findValueWithName(formData, 'company');
-      var city = findValueWithName(formData, '00N580000088gDR');
-
-      analytics.track('Company Interest Sign Up', {
-        name: name,
-        email: email,
-        phone: phone,
-        company: company,
-        city: city,
-        signup_url: location.href,
-        locale: CURRENT_LOCALE
-      });
+      gaTracking(companyCategory, 'submit', 'Company Interest Sign Up');
     });
   }
 
   function trackNewcomerRegPopupOpen() {
     $('.cd-popup-trigger-newcomer-signup').on('click', function() {
-      analytics.track('Newcomer reg popup open', {
-        locale: CURRENT_LOCALE
-      });
+      gaTracking(
+        registrationCategory,
+        'click',
+        'Newcomer reg popup open');
     });
   }
 
   function trackCompanyRegPopupOpen() {
     $('.cd-popup-trigger-signin').on('click', function() {
-      analytics.track('Company reg popup open', {
-        locale: CURRENT_LOCALE
-      });
+      gaTracking(
+        registrationCategory,
+        'click',
+        'Company reg popup open');
     });
   }
 
   function initTrackOfDOMElements() {
     $('[data-track]').each(function() {
       var $element = $(this);
-      var eventName = $element.attr('data-track') || 'click';
+      var category =  $element.attr('category');
+      var action = $element.attr('data-track') || 'click';
```

Comment:
I know it was called `data-track` before this PR, but maybe it would be clearer to rename it `data-action`. (Just a thought.) 

What do you think?